apps/src/templates/projects/PublicGallery.jsx

```diff
@@ -66,31 +67,24 @@ class PublicGallery extends Component {
           limitedGallery={limitedGallery}
           includeDanceParty={includeDanceParty}
         />
-        <div style={styles.clear} />
-        <div style={styles.linkBox}>
-          <a
+        <div style={styles.bottomButton}>
+          <Button
+            __useDeprecatedTag
             href="https://support.code.org/hc/en-us/articles/360001143952"
-            style={styles.link}
-          >
-            <h3>{i18n.reportAbuse()}</h3>
-          </a>
+            color={Button.ButtonColor.gray}
```

Comment:
Also, this seems like a default button, but is there enough contrast between these two shades of grey for it to be accessible?

Reply:
(For posterity: This passes AA but not AAA. If we used 'dark_slate_grey' instead of 'charcoal' it would pass!)

Reply:
Confirmed that it meets AA standards